<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>971</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="StartPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Royal Game of UR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>79</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rulesButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>79</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>RULES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>79</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="GamePage">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="rollButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>ROLL DICE </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveButton">
        <property name="text">
         <string>move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="diceLabel">
        <property name="text">
         <string>    DICE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="boardWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <widget class="QLabel" name="tile_5">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tile_4">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tile_3">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tile_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tile_1">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tile_0">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tile_7">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tile_9">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tile_6">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="statusLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>953</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>  STATUS</string>
         </property>
        </widget>
        <widget class="QPushButton" name="quitButton">
         <property name="geometry">
          <rect>
           <x>562</x>
           <y>3</y>
           <width>391</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>quit</string>
         </property>
        </widget>
        <widget class="QLabel" name="tile_8">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>90</y>
           <width>49</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="ResultPage">
     <widget class="QLabel" name="winnerLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>    WINNER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playAgainButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>79</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>play again</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>79</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RulesPage">
     <widget class="QLabel" name="rulesLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string> RULES</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="rulesBackButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>79</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>321</width>
        <height>192</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Royal Game of Ur â€” Rules&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1. Each player starts at tile 0.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2. On your turn, roll four binary dice (each die is 0 or 1).&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3. Move your piece forward by the total rolled.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4. If you roll 0, your turn ends immediately.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;5. Players take turns.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;6. The first player to reach the final tile wins the game.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
